walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenId4VCIRequestHandler.kt

```diff
@@ -122,8 +125,10 @@ internal class OpenId4VCIRequestHandler(
             credentialMetadata.transformLocalizedIssuerDisplayDefinitionToRequesterStyle()
         val verifiedIdStyle =
             credentialConfiguration.getVerifiedIdStyleInPreferredLocale(requesterStyle.name)
+
+        // By this point, we know credential issuer is not null.
         val accessTokenEndpoint = fetchAccessTokenEndpointFromOpenIdWellKnownConfig(
-            credentialMetadata.credential_issuer ?: ""
+            credentialMetadata.credentialIssuer!!
```

Comment:
If credential issuer is missing, we have to fail isn't it? I think an exception is thrown in CredentialMetadata class in validate method which is missing here. Instead of !!, we could throw the same exception here.